src/components/presentationals/Navbar/Navbar.jsx

```diff
@@ -24,13 +24,16 @@ const Navbar = ({ url, className }) => {
             </Dropdown>
           </div>
         )}
+        {children && children}
       </div>
     </div>
   );
 };
 
 Navbar.propTypes = {
+  children: PropTypes.any,
   className: PropTypes.string,
+  profileUrl: PropTypes.string,
```

Comment:
I think this prop is redundant, `url` is doing what is suppose to do, so lets remove this prop.

Reply:
Done